README.md

```diff
@@ -1,16 +1,67 @@
-# Title
+# Iterator Drill: Find
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Define how the `filter()` method works
+* Demonstrate `filter`
 
 ## Introduction
 
-## SWBAT 1
+Filtering is a very common function of programming that we've all utilized at
+some point in our lives. When shopping for clothing online, chances are, we've
+filtered by sizes, so we can weed out the clothing that we definitely can't fit
+into, and browse through what caters to our needs. If you've ever ordered food
+online, chances are, we've all filtered by what's open right now, and what's
+available for delivery in your neighborhood--filtering out results for restaurants
+that don't deliver to your address. It is a very common action to need to perform.
+This used to be quite annoying to do manually, but now there is no need to fret--
+`filter` is here to help!
```

Comment:
The introduction does a great job of giving end-user examples of filtering. I would recommend smoothing out the wording a bit. If we dropped the "programming" reference at the beginning, we could start with the end-user examples and then connect it to programming, which is the other side of the experience. For example: "Chances are we've all had the experience of filtering a selection of choices online. When shopping for clothing, we might have selected a single size so that we can consider only the search results we need. Or when ordering food online, we could have customized our results to only show restaurants currently open or offering delivery in our neighborhoods. Since it's such a common need for an user, it's also a common function for a web programmer to implement. There are many ways to filter in programming, but the easiest is to use `filter`."